kopi/assetlist.json

```diff
@@ -165,19 +165,32 @@
       "display": "cUSDT.inj",
       "symbol": "CUSDT.inj",
       "logo_URIs": {
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/cusdt.svg"
       },
       "images": [
         {
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/cusdt.svg"
         }
       ],
       "keywords": [
         "dex",
         "mm"
       ],
-      "type_asset": "sdk.coin"
-     
+      "type_asset": "sdk.coin",
+     "traces": [
+        {
+          "type": "ibc",
```

Comment:
There's something different going on here than just a simple IBC transfer. IBC tokens must be type `ics20`, but this is `sdk.coin`. Perhaps if this follows the same price and the convertible back into USDC, this may be a wrapped version of USDC.inj from Kopi (i.e., of  "ibc/ibc/D8A36AE90F20FE4843A8D249B1BCF0CCDDE35C4B605C8DED57BED20C639162D0").
Or perhaps it's a type: "synthetic" version of the peggy asset referenced here.